lib/backburner/queue.ts

```diff
@@ -82,7 +82,7 @@ export default class Queue {
       errorRecordedForStack = queueItems[i + 3]; // Debugging assistance
 
       // method could have been nullified / canceled during flush
-      if (method) {
```

Comment:
thoughts on `typeof method === 'function'` it should cover the `null` case, but test for exactly what we want.